src/index.html

```diff
@@ -7,8 +7,53 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header-nav">
+      <a href="#home">
+        <img 
+          class="nav-icon" 
+          src="images/logo.png" 
+          alt="icon"
+        >
+      </a>
+      <nav class="nav-block">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a class="nav-link is-active" href="#apple">Apple</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#samsung">Samsung</a>
+          </li>
+          <li class="nav-item">
```

Comment:
```suggestion
      </a>
      
      <nav class="nav-block">
        <ul class="nav-list">
          <li class="nav-item">
            <a class="nav-link is-active" href="#apple">
              Apple
            </a>
          </li>
          
          <li class="nav-item">
            <a class="nav-link" href="#samsung">
              Samsung
            </a>
          </li>
          
          <li class="nav-item">
```
fix code style for the whole page